src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -30,8 +30,8 @@
     public static final String VALID_NAME_BOB = "Bob Choo";
```

Comment:
Should the `VALID_NAME` field be renamed with the pet's name for all test cases? Since our address book now stores information about pets.

Same for the other Test utility classes with dummy test case data.

Reply:
'VALID_NAME' field refers to the name of the pet. 'VALID_OWNER_NAME' field refers to the name of pet owner. Refer to line *33* for an example.